docs/upgrades/automated.md

```diff
@@ -129,3 +129,40 @@ kubectl -n system-upgrade get plans -o yaml
 kubectl -n system-upgrade get jobs -o yaml
 ```
 
+## Warning
+
+Downgrading your cluster was never supported and since versions below k3s-upgrade will cordon your nodes when downgrade is attempted:
```

Comment:
Please use the admonitions style we have for all other version gate notices. https://github.com/k3s-io/docs/blob/main/docs/cli/secrets-encrypt.md?plain=1#L11-L13